src/electionguard/ballot.py

```diff
@@ -165,8 +165,8 @@ class CiphertextBallotSelection(ElectionObjectBase, CryptoHashCheckable):
     # The SelectionDescription hash
     description_hash: ElementModQ
 
-    # The encrypted representation of the plaintext field
-    message: ElGamalCiphertext
+    # The encrypted representation of the vote field
+    encrypted_data: ElGamalCiphertext
```

Comment:
A hidden issue here is `CiphertextSelection` aka a union class above this level. It forces both the `CiphertextBallotSelection` and `CiphertextTallySelection` to have the encrypted text use the same variable name. It is not called out and I think it's slightly dangerous for future refactors.

Maybe refactoring them both to inherit from an abstract class with the singular `encrypted_data` field would be better? With the use of dataclass this would change all the parameter ordering above it.

Reply:
we could use a protocol instead to enforce the naming convention on any object type